pkgs/by-name/im/imgbrd-grabber/package.nix

```diff
@@ -1,77 +1,72 @@
-{ lib
-, stdenv
-, cmake
-, fetchFromGitHub
-, wrapQtAppsHook
-, qtmultimedia
-, qttools
-, qtdeclarative
-, qtnetworkauth
-, qtbase
-, makeWrapper
-, catch2
-, nodejs
-, libpulseaudio
-, openssl
-, rsync
-, typescript
+{
+  lib,
+  stdenv,
+  cmake,
+  fetchFromGitHub,
+  makeWrapper,
+  catch2,
+  nodejs,
+  libpulseaudio,
+  openssl,
+  rsync,
+  typescript,
+  qt6,
 }:
-
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "imgbrd-grabber";
-  version = "7.10.0";
+  version = "7.12.2";
 
   src = fetchFromGitHub {
     owner = "Bionus";
     repo = "imgbrd-grabber";
-    rev = "v${version}";
-    sha256 = "sha256-AT6pN2do0LlH6xAXKcFQv+oderD88/EiG1JnCw6kOOg=";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-6XfIaASfbvdPovtdDEJtsk4pEL4Dhmyq8ml4X7KZ4DE=";
     fetchSubmodules = true;
   };
 
   buildInputs = [
     openssl
     libpulseaudio
     typescript
-  ];
-
-  nativeBuildInputs = [
+    qt6.wrapQtAppsHook
     makeWrapper
-    qtmultimedia
-    qtbase
-    qtdeclarative
-    qttools
-    qtnetworkauth
     nodejs
-    cmake
-    wrapQtAppsHook
   ];
 
+  nativeBuildInputs =
+    with qt6;
+    [
+      qtbase
+      qtdeclarative
+      qttools
+      qtnetworkauth
+      qtmultimedia
+    ]
```

Comment:
Those are still not correct and should go to buildInputs.

Reply:
Applied suggestion. Let me know if there are other changes to make